alf/algorithms/predictive_representation_learner.py

```diff
@@ -265,6 +266,44 @@ def rollout_step(self, time_step: TimeStep, state):
         latent, state = self._encoding_net(time_step.observation, state)
         return AlgStep(output=latent, state=state)
 
+    def predict_multi_step(self, init_rep, actions, state=None):
+        """Perform multi-step predictions based on the initial latent
+            representation and actions sequences.
+        Args:
```

Comment:
Comment that the target of the current step is not predicted.

Reply:
Now made it consistent within PredictiveRepresentationLearner and always include the current step. Any special post processing such as removing the current step is left for the caller.